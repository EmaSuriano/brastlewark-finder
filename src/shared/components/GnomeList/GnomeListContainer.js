import styled from 'styled-components';

const GnomeListContainer = styled.div`
  display: grid;
  grid-gap: 30px;
  padding-bottom: 30px; // end of the result
  grid-template-columns: repeat(auto-fill, minmax(250px, 1fr));
  grid-auto-rows: 160px;
  justify-self: stretch;
  align-content: center;
`;

export default GnomeListContainer;
