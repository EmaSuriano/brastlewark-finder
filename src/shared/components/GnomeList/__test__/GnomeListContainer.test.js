import React from 'react';
import { shallow } from 'enzyme';
import GnomeListContainer from '../GnomeListContainer';

describe('<GnomeListContainer />', () => {
  it('render', () => {
    expect(shallow(<GnomeListContainer />)).toMatchSnapshot();
  });
});
