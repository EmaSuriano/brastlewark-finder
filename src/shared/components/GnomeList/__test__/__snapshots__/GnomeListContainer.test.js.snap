// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GnomeListContainer /> render 1`] = `
.c0 {
  display: grid;
  grid-gap: 30px;
  padding-bottom: 30px;
  grid-template-columns: repeat(auto-fill,minmax(250px,1fr));
  grid-auto-rows: 160px;
  justify-self: stretch;
  -webkit-align-content: center;
  -ms-flex-line-pack: center;
  align-content: center;
}

<div
  className="c0"
/>
`;
